src/Serialization/upgrades/0.11.2.jl

```diff
@@ -0,0 +1,88 @@
+################################################################################
+# Upgrade Summary
+# Defines an OscarBasictype, and any Vector{OscarBasictype} will not have
+# backrefs in any entry. Instead all entries of the array will have their
+# representative only and the entry_type is given for all entries once on
+# the save level as the :vector key
+
+upgrade_script = UpgradeScript(
```

Comment:
This defines an untype global. I'd recommend we avoid this and instead directly push, like so:
```suggestion
push!(upgrade_scripts, UpgradeScript(
```
... and then of course the end of this file also needs to be adjusted